src/css/main.css

```diff
@@ -4,6 +4,7 @@
  */
 
 @import url('/src/css/partials/variables.css');
+@import url(/src/css/partials/adventages.css);
```

Comment:
Swój import umieść pod resetem, bo kolejność ma znaczenie i jak Twoje są na górze to reset, może ( ale nie musi ) mieszać Ci w kodzie

Reply:
Nie mogę umieścić pod resetem bo wywala mi błąd że wszystkie @import muszą być umieszczone bezpośrednio pod sobą i wgl się strona nie ładuje.

Reply:
Okey, nie wiedziałem o tym. Zapytam Ivana co i jak. Możliwe że reset będę musiał wrzucić w osobny plik a nie w main jak teraz. Zobaczymy ;) Kod działa i będzie szedł pod merge 👍